### PR TITLE
Fix/adjust order flow  cleaning modal

### PR DESCRIPTION
**Pull request by zoeGuava**
- [x] adjust the function removeOrderItem
- [x] adjust the display text for the discount in the promotion modal
- [x] add the cleaning check modal component
- [x] add reminder to told user can switch to the dining table number

### CODE REVIEW

File: src/apps/Order/components/Modals/Clean.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const { table } = useStore();
+
+  const { mutateAsync } = usePatchTable();
+
+  const handleCleanModal = async () => {
+    const res = await mutateAsync({ id: table.id, status: TableStatus.IDLE, customerNum: 0 });
+
+    if (res.status === HttpStatusCode.Ok) {
+      setIsOpen(false);
+    }
```
Comment (alphatero) [inline src/apps/Order/components/Modals/Clean.tsx]: 這層別放 api 層的判斷

**Review by alphatero — COMMENTED** [review]
你要不要順便加個 loading 順便解一下衝突

**Comment by zoeGuava** [comment]
- [ ] return 'success' from patchTable api
- [ ] when remove meal, use isLoading state in order/Modal/LogItem to switch display layout

@yuyu0905 不好意思...
[1f683ce](https://github.com/alphatero/siau-kha-frontend/pull/58/commits/1f683ce5075fd78d8ec85efa913d262b0147e27f)：這邊有動到你寫的 api 回傳的值
[432ee5b](https://github.com/alphatero/siau-kha-frontend/pull/58/commits/432ee5b17781173f449c0170fab13ded5b2a5866)：另外因為這邊沒改的話，deploy 會失敗報錯，所以也有修改了你原本用 HttpStatusCode 來驗證是否呼叫成功的寫法
抱歉改完才發現動到你的 code...

**Comment by yuyu0905** [comment]
更動 code 很常見阿~
有問題再麻煩改~感謝!

**Review by alphatero — APPROVED** [review]

**Pull request merged**